src/features/changeForwarding.ts

```diff
@@ -6,40 +6,24 @@
 import { Uri, workspace } from 'vscode';
 import { OmniSharpServer } from '../omnisharp/server';
 import * as serverUtils from '../omnisharp/utils';
-import { FileChangeType, LinePositionSpanTextChange } from '../omnisharp/protocol';
+import { FileChangeType } from '../omnisharp/protocol';
 import { IDisposable } from '../Disposable';
 import CompositeDisposable from '../CompositeDisposable';
 
 function forwardDocumentChanges(server: OmniSharpServer): IDisposable {
 
     return workspace.onDidChangeTextDocument(event => {
 
-        let { document, contentChanges } = event;
+        let { document } = event;
         if (document.isUntitled || document.languageId !== 'csharp' || document.uri.scheme !== 'file') {
             return;
         }
 
```

Comment:
I think we should restore this change since there are no changes to be synced.

```suggestion

        if (contentChanges.length === 0) {
            // This callback fires with no changes when a document's state changes between "clean" and "dirty".
            return;
        }

```